### DISCUSSION TITLE
Deposits can be renamed to something other than the item they produce

### DISCUSSION THREAD

**Discussion by goldenarrow** [voxeltycoon/issues#441] [Modding] [answered]
So, still with my farm deposit (see https://github.com/voxeltycoon/issues/discussions/440), it'd be nice if I could produce milk churns from it. But at the moment if I do that, it calls the deposit "Milk churns". 

I've had to rename the milk churn item to "Dairy Products" so it doesn't look weird having a grassy area called Milk Churns.

**Comment by andrewpey** [comment, marked as answer]
Done in [0.82](https://voxeltycoon.xyz/changelog#0.82)
